package.json

```diff
@@ -1,34 +1,45 @@
 {
   "name": "theme-colors",
   "version": "0.0.5",
-  "description": "easily generate color shades for themes",
+  "description": "Easily generate color shades for themes",
+  "repository": "unjs/theme-colors",
+  "license": "MIT",
   "keywords": [
     "theme",
     "colors",
     "shades",
     "tailwind",
     "tailwindcss"
   ],
-  "repository": "nuxt-contrib/theme-colors",
-  "license": "MIT",
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
```

Comment:
Could you update this part of the package?

Here an exemple:
```json
"exports": {
    ".": {
      "import": {
        "types": "./dist/index.d.mts",
        "default": "./dist/index.mjs"
      },
      "require": {
        "types": "./dist/index.d.cts",
        "default": "./dist/index.cjs"
      }
    }
  },
  "main": "./dist/index.cjs",
  "types": "./dist/index.d.ts",
```

Reply:
Sure. I have update to the latest UnJS repo template, which slightly differs. But I guess to prefer the template is better than customizing it (although I might prefer the syntax you posted).